.vscode/launch.json

```diff
@@ -619,6 +620,24 @@
       },
       "preLaunchTask": "Compile",
       "internalConsoleOptions": "openOnSessionStart"
+    },
+    {
+        "name": "Debug Faux Jest Unit Tests",
+        "cwd": "${workspaceRoot}/packages/salesforcedx-sobjects-faux-generator",
+        "type": "node",
+        "request": "launch",
+        "runtimeArgs": ["--inspect-brk", "${workspaceRoot}/packages/salesforcedx-sobjects-faux-generator/node_modules/.bin/jest", "--runInBand", "--coverage", "false"],
+        "console": "integratedTerminal",
+        "internalConsoleOptions": "neverOpen"
+    }, 
+    {
+      "name": "Debug Apex Debug Jest Unit Tests",
+      "cwd": "${workspaceRoot}/packages/salesforcedx-apex-debugger",
+      "type": "node",
+      "request": "launch",
+      "runtimeArgs": ["--inspect-brk", "${workspaceRoot}/packages/salesforcedx-apex-debugger/node_modules/.bin/jest", "--runInBand", "--coverage", "false"],
+      "console": "integratedTerminal",
+      "internalConsoleOptions": "neverOpen"
```

Comment:
I added these as examples for how to run jest tests in vscode.  I ended up installing an extension that lets. me just interact with test files and no longer found these overly useful.